### PR TITLE
napatech: minor fixes in the Napatech code

### PR DESCRIPTION
**Pull request by tourko**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [ x] I have read the contributing guide lines at https://suricata.readthedocs.io/en/latest/devguide/codebase/contributing/contribution-process.html
- [ x] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [ x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Describe changes:
- Added 'module' argument to the SCLog() functions used by source-napatech.c
- Set SCTime_t structure from the HW timestamps
- Added return value check for ByteExtractStringUint8() calls
- Set THV_RUNNING flag for the TmThreads 

Replaces [#8802](https://github.com/OISF/suricata/pull/8802)

### CODE REVIEW

**Comment by victorjulien** [comment]
Git history is still unclean. The PR needs to have only code commits, no merge commits.

Additionally, you can reword things to not mention 7.0.0 (which isn't out yet), just describe in the message: why & how.

**Comment by victorjulien** [comment]
replaces #8857 

Please add a reference like above in a PR replacing an old PR.

File: src/util-napatech.c [hunk 1]
```diff
@@ -643,6 +644,7 @@ static void *NapatechStatsLoop(void *arg)
     }
 
     TmThreadsSetFlag(tv, THV_INIT_DONE);
```
Comment (jlucovsky) [inline src/util-napatech.c:646]: I think that this call can be removed but you'd have to test it. See `source-af-packet` (and other packet source modules for examples).

File: src/source-napatech.c [hunk 2]
```diff
@@ -26,6 +26,7 @@
  *
  */
 #include "suricata-common.h"
+#include "action-globals.h"
```
[the remark this answers is not in the document]
Comment (tourko) [reply src/source-napatech.c:29]: It is require for ACTION_DROP definition in the PacketCheckAction(p, ACTION_DROP).

**Comment by tourko** [comment]
I have created a new PR#8870 with the suggested changes.

**Pull request closed without merging**